system-tests/lib/crecli/por/por.go

```diff
@@ -5,42 +5,67 @@ import (
 	"os"
 	"strings"
 
-	"github.com/ethereum/go-ethereum/common"
 	"github.com/pkg/errors"
-
-	libcrecli "github.com/smartcontractkit/chainlink/system-tests/lib/crecli"
 )
 
-func CreateConfigFile(feedsConsumerAddress common.Address, feedID, dataURL, writeTargetName string, authKeySecretName *string) (*os.File, error) {
-	configFile, err := os.CreateTemp("", "config.json")
+type Config interface {
+	FeedIDGetter
+	FeedIDSetter
+}
+
+type FeedIDGetter interface {
+	GetFeedID() string
+}
+
+type FeedIDSetter interface {
+	SetFeedID(string)
+}
```

Comment:
why do we need to use interfaces?

Reply:
so that I can reuse `CreateConfigFile` to use any type of config that implements updating the FeedID